mkdocs/overrides/devbook/transactions/messages.py

```diff
@@ -0,0 +1,232 @@
+import json
+import os
+import time
+import urllib.error
+import urllib.request
+from binascii import hexlify
+
+from symbolchain.CryptoTypes import PrivateKey, PublicKey
+from symbolchain.facade.SymbolFacade import SymbolFacade
+from symbolchain.symbol.MessageEncoder import MessageEncoder
+from symbolchain.symbol.Network import NetworkTimestamp
+from symbolchain.sc import Amount
+
+# Configuration
+NODE_URL = os.environ.get(
+	"NODE_URL", "https://001-sai-dual.symboltest.net:3001"
+)
+
+
+# Helper function to poll for confirmed transaction
+def wait_for_confirmation(hash_value, label):
+	print(f"Polling for {label} confirmation...")
+	confirmed = False
+	attempts = 0
+	max_attempts = 60
+
+	while not confirmed and attempts < max_attempts:
+		try:
+			url = f"{NODE_URL}/transactions/confirmed/{hash_value}"
+			with urllib.request.urlopen(url) as response:
+				confirmed = True
+				print(f"  {label} confirmed!")
+				return json.loads(response.read().decode())
+		except urllib.error.HTTPError:
+			# Transaction not yet confirmed
+			pass
+		attempts += 1
+		time.sleep(2)
+
+	if not confirmed:
+		raise Exception(
+			f"{label} not confirmed after {max_attempts} attempts"
+		)
+
+
+print(f"Using node {NODE_URL}")
+
+# Set up sender and recipient accounts
+facade = SymbolFacade("testnet")
+
+sender_private_key_string = os.environ.get(
+	"SENDER_PRIVATE_KEY",
+	"0000000000000000000000000000000000000000000000000000000000000000",
+)
+sender_key_pair = facade.KeyPair(
+	PrivateKey(sender_private_key_string)
+)
+sender_address = facade.network.public_key_to_address(
+	sender_key_pair.public_key
+)
+
+recipient_private_key_string = os.environ.get(
+	"RECIPIENT_PRIVATE_KEY",
+	"1111111111111111111111111111111111111111111111111111111111111111",
+)
+recipient_key_pair = facade.KeyPair(
+	PrivateKey(recipient_private_key_string)
+)
+recipient_address = facade.network.public_key_to_address(
+	recipient_key_pair.public_key
+)
+
+print(f"Sender address: {sender_address}")
+print(f"Recipient address: {recipient_address}\n")
+
+# ===== PLAIN TEXT MESSAGE =====
+print("=== Sending Plain Text Message ===")
+
+# Fetch current network time
+time_path = "/node/time"
+print(f"Fetching current network time from {time_path}")
+with urllib.request.urlopen(f"{NODE_URL}{time_path}") as response:
+	response_json = json.loads(response.read().decode())
+	timestamp = NetworkTimestamp(int(
+		response_json['communicationTimestamps']['receiveTimestamp'])
+	)
+	print(f"  Network time: {timestamp.timestamp} ms since nemesis")
+
+# Fetch recommended fees
+fee_path = "/network/fees/transaction"
+print(f"Fetching recommended fees from {fee_path}")
+with urllib.request.urlopen(f"{NODE_URL}{fee_path}") as response:
+	response_json = json.loads(response.read().decode())
+	median_mult = response_json["medianFeeMultiplier"]
+	minimum_mult = response_json["minFeeMultiplier"]
+	fee_mult = max(median_mult, minimum_mult)
+	print(f"  Fee multiplier: {fee_mult}")
```

Comment:
Since this bit is common for plain and encrypted messages, I would take it out of the Plain section and put it right after printing the addresses.